test_view_model.py

```diff
@@ -0,0 +1,123 @@
+from todo_item import TodoItem
+from view_model import ViewModel
+import arrow
+
+def test_to_do():
+    # Passing through blank value for the Date Input as not required for this test
+    items = [
+        TodoItem(1, "Started Todo", "Not Started", ""),
+        TodoItem(2, "Doing Todo", "In Progress", ""),
+        TodoItem(3, "Done Todo", "Completed", ""),
+    ]
+
+    frank = ViewModel(items)
+
+    # Act
+    # [TodoItem(1, "Started Todo", "Not Started")]
+    todo_items = frank.to_do
+
+    # Assert
+    assert len(todo_items) == 1
+
+    todo_item = todo_items[0]
+
+    assert todo_item.status == "Not Started"
+
+def test_doing():
+    # Arrange
+    items = [
+        TodoItem(1, "Started Todo", "Not Started", ""),
+        TodoItem(2, "Doing Todo", "In Progress", ""),
+        TodoItem(3, "Done Todo", "Completed", ""),
+    ]
+
+    vader = ViewModel(items)
```

Comment:
Why's your view model called `vader`? 😛 

I assume you were just experimenting with variable names, but it would be clearer to call it e.g. `view_model`